### ISSUE TITLE
dfIPA missing upstream and downstreamss columns 

### ISSUE THREAD

**Issue by mkatsanto** [RJWANGbioinfo/APAlyzer#7]
Hi! I am currently trying to apply your method by creating a reference annotation from an ensembl gtf file.
I followed your example in the documentation and somehow the dfIPA dataframe misses the upstream and downstreamSS columns.  I am using a version tagged as 1.6 from bioconductor.  https://bioconductor.org/packages/release/bioc/html/APAlyzer.html .
I also built a docker image (https://hub.docker.com/repository/docker/mkatsanto/apalyzer , test2 tag) if you want to reproduce the error. 

The code that produces this error is the following:
 ```
## build Reference ranges for 3'UTR PASs in mouse
download.file(url='ftp://ftp.ensembl.org/pub/release-99/gtf/mus_musculus/Mus_musculus.GRCm38.99.gtf.gz',
		  destfile='Mus_musculus.GRCm38.99.gtf.gz')			  
GTFfile="Mus_musculus.GRCm38.99.gtf.gz"	
PASREFraw=PAS2GEF(GTFfile)	
refUTRraw=PASREFraw$refUTRraw
dfIPAraw=PASREFraw$dfIPA
dfLEraw=PASREFraw$dfLE
PASREF=REF4PAS(refUTRraw,dfIPAraw,dfLEraw)
```

Thanks for any help!

**Comment by RJWANGbioinfo** [comment]
Hi @mkatsanto,
Thanks for your feedback. This issue has been fixed in version 1.7.1
Please update your APAlyzer and give it a try through:
```
BiocManager::install('RJWANGbioinfo/APAlyzer')
```

**Comment by RJWANGbioinfo** [comment]
Close this ticket since it has been fixed by version 1.7.1
